scripts/pages/index.page.js

```diff
@@ -105,6 +106,38 @@ const scrollIntro = () => {
   }
 };
 
+const newsletterSubscribe = (e) => {
+  e.preventDefault();
+  const form = document.querySelector('.c-newsletter-form');
+
+  if (form.checkValidity() === false) {
+    return;
```

Comment:
Some user feedback would be nice (reuse stuff used in server response handling)

Reply:
browser handles that automatically, highlights the field in red.